client/sites-dashboard-v2/style.scss

```diff
@@ -42,7 +42,6 @@
 		.site-monitoring-overview {
 			overflow-y: initial;
 			max-height: initial;
-			padding-bottom: 88px;
```

Comment:
Can we make this change in dotcom-style.scss?

Reply:
I think this is dotcom styles so it's fine to change it here. But maybe we have to move all of related styles to dotcom-styles.scss.